Makefile

```diff
@@ -3,6 +3,7 @@ PROJECT = landing
 COMMANDS = api
 CODECOV_TOKEN ?=
 DESTINATION ?= public
+DOCKERFILES = Dockerfile:landing Dockerfile.api:landing-api
```

Comment:
Yep. Right now the makefile does not even build the slackin, we might have to do that too. I will do it in another PR.